fuel/datasets/base.py

```diff
@@ -28,10 +29,6 @@ class Dataset(object):
         'targets')`` for MNIST (regardless of which data the data stream
         actually requests). Any implementation of a dataset should set this
         attribute on the class (or at least before calling ``super``).
-    default_iteration_scheme : :class:`.IterationScheme`, optional
```

Comment:
Okay, I am 100% happy to have `get_example_stream` instead. Would you prefer to make it a part of this PR or of a later? 

```python
class ExampleScheme(BatchScheme):

    def __init__(self, num_examples):
        super(ExampleScheme, self).__init__(num_examples, 1)

    def get_request_iterator(self):
        return _iter(range(self.num_examples))
```